api/v1alpha1/podhttpchaos_types.go

```diff
@@ -182,6 +182,8 @@ const (
 )
 
 // +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name="action",type=string,JSONPath=`.spec.action`
+// +kubebuilder:printcolumn:name="duration",type=string,JSONPath=`.spec.duration`
```

Comment:
We do not need add this feature for `PodXXXChaos`

Reply:
OK